src/applications/gqrx/mainwindow.cpp

```diff
@@ -668,8 +670,8 @@ void MainWindow::updateFrequencyRange(bool ignore_limits)
         qDebug() << QString("New frequnecy range: %1 - %2 MHz (step is %3 Hz but we use 1 Hz).").
                     arg(startd*1.0e-6).arg(stopd*1.0e-6).arg(stepd);
 
-        qint64 start = (qint64)startd + d_lnb_lo;
-        qint64 stop  = (qint64)stopd  + d_lnb_lo;
+        qint64 start = (qint64)(startd + rx->get_filter_offset()) + d_lnb_lo;
+        qint64 stop  = (qint64)(stopd + rx->get_filter_offset())  + d_lnb_lo;
```

Comment:
This was actually my first ideas as well. But this on its own would just allow setting a hardware frequency outside of the limits.

Currently, changing the main frequency dial changes the hardware frequency. To change the offset one has to use the designated dial or to click into the spectrum. If we allow full access to the whole spectrum through the main dial we would have to implement some means of automatically changing the offset when the chosen frequency is outside of the hardware limits. This would of course make this patch more invasive but in particular it would make the behavior of the main dial less predictable.

So I chose to keep the current behavior: To deviate from the hardware frequency one has to enter an offset frequency or click into the spectrum. Changing the main dial does not interfere with the offset in any way, so its min and max values have to depend on the currently set offset.